build/configs/imxrt1020-evk/loadable_elf_apps/defconfig

```diff
@@ -395,9 +395,9 @@ CONFIG_ARCH_USE_FLASH=y
 #
 CONFIG_FLASH_PARTITION=y
 CONFIG_FLASH_MINOR=0
-CONFIG_FLASH_PART_SIZE="2048,512,3072,128,128,"
-CONFIG_FLASH_PART_TYPE="kernel,none,smartfs,ftl,config,"
-CONFIG_FLASH_PART_NAME="kernel,app,userfs,ftl,config,"
```

Comment:
We should consider there is no common binary. In this case, we should set the user binary partition.
How can we set this?

Reply:
If there is no common binary, then we need to add the user binary partition manually. I dont think we can automate it.